tests/ui/panics/short-ice-remove-middle-frames.run.stderr

```diff
@@ -0,0 +1,12 @@
+thread 'main' panicked at 'debug!!!', $DIR/short-ice-remove-middle-frames.rs:52:5
+stack backtrace:
+   0: std::panicking::begin_panic
+   1: short_ice_remove_middle_frames::seven
+   2: short_ice_remove_middle_frames::sixth
+   3: short_ice_remove_middle_frames::fifth::{{closure}}
+   4: short_ice_remove_middle_frames::second
```

Comment:
Can you please emit some kind of marker to indicate that frames have been omitted? This is just plain confusing. For example

```suggestion
   [some frames omitted]
   4: short_ice_remove_middle_frames::second
```
or

```suggestion
   [2 frames omitted]
   4: short_ice_remove_middle_frames::second
```

Reply:
I made a change: https://github.com/rust-lang/rust/commit/ddacb2dbb91ac1a438ab14db80b9a3457ecba60c

Found out we need to update https://github.com/rust-lang/backtrace-rs.git to add an API to emit the message.

Maybe we can enhance it in another PR?

Reply:
Could you open an issue for adding the "N frame omitted" line, please?

Reply:
Sure, I will create an issue, then make a PR for backtrace-rs first, then another follow-up PR for it.